int/int_test.go

```diff
@@ -18,15 +18,14 @@ var (
 )
 
 const (
-	MaxWaitTime time.Duration = 30 * time.Second
+	MaxWaitTime = 30 * time.Second
 )
 
 var _ = BeforeSuite(func() {
 	var err error
 	i, err = NewIntegration()
 	Expect(err).NotTo(HaveOccurred())
 	err = i.DisableWebhook()
-	Expect(err).NotTo(HaveOccurred())
```

Comment:
Do we need to undo these changes to the integration tests since they were just symptoms and not the actual problems before we commit this?

Reply:
I don't think we _need_ to, but we should.  I'll roll some of these back.